application/config.json.template

```diff
@@ -89,7 +89,11 @@
         ],
         "isHostSimilarToKeywordDistanceThreshold": 2,
         "suspiciousAttachmentsThreshold": 3,
-        "suspiciousAttachmentNamePattern": "(image|\\d{1,2})\\.[^.]{0,5}"
+        "suspiciousAttachmentNamePattern": "(image|\\d{1,2})\\.[^.]{0,5}",
+        "maxAllowedSimilarMessages": 2,
+        "similarMessagesWindow": 1,
+        "similarMessageLengthIgnore": 10,
+        "similarMessagesWhitelist": []
```

Comment:
i dont think we need that, please remove.

the UX for it for maintainers would be really weird, having to bloat the config with 100 character long messages and whatnot. and then it didnt work bc of a smiley or other stuff that ends up slightly different.

Reply:
sorry, the preview didnt turn out the way i wanted. i was referring to the message whitelist.